Terasology/GooKeeper
# fix: NPE in `EconomySystem`

# How to test

1. Start a `GooKeeper` game without this PR
2. Spawn a visitor with `spawnPrefab visitor`
3. Observe that game crashing
4. Repeat (1) and (2) with this PR and observe that the game doesn't crash and the visitor is spawned correctly (visitor will likely be invisible without changes from https://github.com/Terasology/GooKeeper/pull/27 being present).
